vnext/PropertySheets/React.Cpp.props

```diff
@@ -27,7 +27,7 @@
     <HERMES_Version Condition="'$(HERMES_Version)' == ''">0.1.6</HERMES_Version>
     <HERMES_Package Condition="'$(HERMES_Package)' == ''">$(SolutionDir)packages\ReactNative.Hermes.Windows.$(HERMES_Version)</HERMES_Package>
 
-    <USE_V8 Condition="'$(USE_V8)' == '' AND '$(Platform)' != 'ARM' AND '$(Platform)' != 'ARM64'">true</USE_V8>
+    <USE_V8 Condition="('$(USE_V8)' == '') Or ('$(Platform)' == 'ARM' Or '$(Platform)' == 'ARM64')">false</USE_V8>
```

Comment:
>false [](start = 100, length = 5)

This is breaking the win32 build (making it incompatible with Office consumption).

Reply:
@tudorms the CI is building Win32, and is passing. What is breaking?

Reply:
Forgot this during triage, but USE_V8 will affect control flow, etc more than the NuGet package. Office is relying on V8 right now, so we need to continue building.

IIRC we'll fallback to Chakra or crash if someone asks for V8 and USE_V8 is off.